tests/framework/CodeInterfaceTests/Serpent/SerpentTest1/sampleInput/testInput.serpent

```diff
@@ -0,0 +1,132 @@
+set title "MSBR, no Th,U-233 8.5E-11, after 400 days 7.75E-1"
```

Comment:
This file is created by RAVEN in the perturbation process. It should not be here. Please remove it